guides/api.md

```diff
@@ -31,7 +31,6 @@
 
 * [ao](#ao)
     * [.logLevel](#ao.logLevel)
-    * [.serviceKey](#ao.serviceKey)
```

Comment:
Is `serviceKey` no longer supported?

Reply:
no, it's not. it's not relevant in a lambda environment and it's not at all clear why a program would ever need it.

Reply:
discussed in slack